pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -193,16 +195,77 @@ func (c *Cluster) Start() error {
 	}
 	logger.Infof("wait timeout for healthy OSDs during upgrade or restart is %q", c.clusterInfo.OsdUpgradeTimeout)
 
+	osdsToSkipReconcile, err := c.getOSDsToSkipReconcile()
+	if err != nil {
+		logger.Warningf("failed to get osds to skip reconcile. %v", err)
+	}
+
 	// prepare for updating existing OSDs
 	updateQueue, deployments, err := c.getOSDUpdateInfo(errs)
 	if err != nil {
 		return errors.Wrapf(err, "failed to get information about currently-running OSD Deployments in namespace %q", namespace)
 	}
-	osdsToSkipReconcile, err := c.getOSDsToSkipReconcile()
-	if err != nil {
-		logger.Warningf("failed to get osds to skip reconcile. %v", err)
+
+	var replaceConfig *OSDReplaceConfig
+	if c.spec.Storage.Store.UpdateStore == OSDStoreUpdateConfirmation {
+		logger.Infof("start migrating osds to backend store %q", c.spec.Storage.Store.Type)
```

Comment:
Yes. I will be logged even after the conversion is completed (unless the `yes-really-update-store` flag is removed). But it will also log that no OSDs require migration since all have been migrated already.

Reply:
This sounds like we are proceeding with some migration. Instead of logging this, can we first detect how many OSDs need conversion, and log that? Or maybe just make this debug level.

Reply:
Makes sense. Removed this log.